ibis/expr/operations/structs.py

```diff
@@ -33,7 +32,7 @@ class StructColumn(Value):
     names: VarTuple[str]
     values: VarTuple[Value]
 
-    shape = ds.columnar
+    shape = rlz.shape_like("values")
```

Comment:
Could you please add a simple operations test for it under `ibis/expr/operations/test_structs.py`?

Reply:
Done.